### ISSUE TITLE
Problems when creating a new cluster - datacenter not found

### ISSUE THREAD

**Issue by mueggelsee** [xetys/hetzner-kube#274]
Is this a bug? I am using https://github.com/xetys/hetzner-kube/releases/download/0.4.1/hetzner-kube-0.4.1-linux-amd64 and get the following error message:
```
hetzner-kube cluster create -w 5 --ssh-key kubernetes
2019/04/18 02:12:32 Creating new cluster

NAME:rowan-lionshimmer
MASTERS: 1
WORKERS: 5
ETCD NODES: 0
HA: false
ISOLATED ETCD: false
2019/04/18 02:12:34 creating server 'rowan-lionshimmer-master-01'...
  --- [======================================] 100%
2019/04/18 02:12:39 Created node 'rowan-lionshimmer-master-01' with IP 159.69.94.236
2019/04/18 02:12:39 creating server 'rowan-lionshimmer-worker-01'...
  --- [======================================] 100%
2019/04/18 02:12:43 Created node 'rowan-lionshimmer-worker-01' with IP 159.69.105.10
2019/04/18 02:12:43 creating server 'rowan-lionshimmer-worker-02'...
  --- [======================================] 100%
2019/04/18 02:12:47 Created node 'rowan-lionshimmer-worker-02' with IP 95.216.168.136
2019/04/18 02:12:47 creating server 'rowan-lionshimmer-worker-03'...
  --- [======================================] 100%
2019/04/18 02:12:56 Created node 'rowan-lionshimmer-worker-03' with IP 159.69.183.201
2019/04/18 02:12:56 creating server 'rowan-lionshimmer-worker-04'...
2019/04/18 02:12:56 datacenter 'fsn1-dc8' not found (invalid_input)
```

**Comment by domains-index** [comment]
Same: 
```
hetzner-kube cluster create --name px-cluster --ssh-key XXXXXX  --master-server-type cx51  --worker-count 4 --worker-server-type cx51
2019/05/16 07:37:41 Creating new cluster

NAME:px-cluster
MASTERS: 1
WORKERS: 4
ETCD NODES: 0
HA: false
ISOLATED ETCD: false
2019/05/16 07:37:42 creating server 'px-cluster-master-01'...
  --- [======================================] 100%
2019/05/16 07:37:48 Created node 'px-cluster-master-01' with IP XX
2019/05/16 07:37:48 creating server 'px-cluster-worker-01'...
  --- [======================================] 100%
2019/05/16 07:37:54 Created node 'px-cluster-worker-01' with IP XXX
2019/05/16 07:37:54 creating server 'px-cluster-worker-02'...
  --- [======================================] 100%
2019/05/16 07:38:01 Created node 'px-cluster-worker-02' with IP XX
2019/05/16 07:38:01 creating server 'px-cluster-worker-03'...
  --- [======================================] 100%
2019/05/16 07:38:08 Created node 'px-cluster-worker-03' with IP XX
2019/05/16 07:38:08 creating server 'px-cluster-worker-04'...
2019/05/16 07:38:09 datacenter 'fsn1-dc8' not found (invalid_input)
```

**Comment by evahteev** [comment]
it looks like 'fsn1-dc8' is no longer in the list from:
https://api.hetzner.cloud/v1/datacenters
So workaround would be to specify --datacenters fsn1-dc14 (or  nbg1-dc3, hel1-dc2)

**Comment by xetys** [comment]
This issue should be solved finally by #290 as we ask now for the data centers using API, if there is a context
